src/Diagnostics/FxCop/Desktop.Analyzers/Core/Desktop.Analyzers.Common.props

```diff
@@ -0,0 +1,25 @@
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <!-- 
+    This file contains the rules that have been implemented in this package and therefore should be disabled for the binary FxCop.
+    When a new rule is implemented in this package, make sure to add an entry below to disable that rule in FxCop to avoid duplicate 
+    issues.
+    The format is -[Category]#[ID]
+  -->
+  <PropertyGroup>
+    <CodeAnalysisRuleSetOverrides>$(CodeAnalysisRuleSetOverrides);
+      -Microsoft.Design#CA1001;
```

Comment:
Seems like all these entries are for the non-Desktop analyzers? You seem to only implement CA2153 in this project?